README.adoc

```diff
@@ -2512,12 +2512,12 @@ Now everything should be ready to start using Fairspace:
 
 Requires:
 
-* Helm >= 3.13.x
-* kubectl >= 1.27.x
+* Helm >= 3.14.x
+* kubectl >= 1.26.x
```

Comment:
why version decrease? On GCP we are on 1.27.8

Reply:
I referred to the compatibility table in  helm's doc :

<img width="744" alt="image" src="https://github.com/thehyve/fairspace/assets/11148506/89ac63c0-c3df-4ba4-a382-d7b153568462">